src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -85,6 +86,12 @@ def load_arguments(self, _):
                    help="Create your Azure Spring Cloud service in an Azure availability zone or not, "
                         "this could only be supported in several regions at the moment ",
                    default=False, is_preview=True)
+        c.argument('build_pool_size',
+                   arg_type=get_enum_type(['S1', 'S2', 'S3', 'S4', 'S5']),
+                   validator=validate_build_pool_size,
+                   default='S1',
+                   is_preview=True,
+                   help='Only support in enterprise tier now. Size of build agent pool. See Azure Spring Cloud Doc for size info.')
```

Comment:
```suggestion
                   help='(Enterprise Tier Only) Size of build agent pool. See Azure Spring Cloud Doc for size info.')
```

Reply:
Do you have a short link for the doc size? You can create a aka.ms short link first and later update the link to the public one.